api/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM python:3.12.4
+
+RUN pip install pipenv
+
+COPY Pipfile* /
+RUN pipenv install --ignore-pipfile
+
+RUN mkdir /app
+COPY . /app
+
+RUN ln -s $(pipenv --venv)/bin/gunicorn
+
+CMD [ "./gunicorn", "-w", "4", "-b", "0.0.0.0:80", "app:api_app" ]
```

The dev may want to use custom parameters, so probably something as follows would be better in that case
```suggestion
ENTRYPOINT [ "./gunicorn" ]
CMD [ "-w", "4", "-b", "0.0.0.0:80", "app:api_app" ]
```